indico/modules/vc/models/vc_rooms.py

```diff
@@ -331,26 +331,38 @@ def get_linked_for_event(cls, event):
         """Get a dict mapping link objects to event vc rooms"""
         return {vcr.link_object: vcr for vcr in cls.find_for_event(event)}
 
-    def delete(self, user):
+    def delete(self, user, delete_all = 0):
```

Comment:
Python has booleans ;) and PEP8 says "no space around the `=` of kwargs" --> `delete_all=False`